src/Kunstmaan/AdminBundle/Form/MediaTokenTransformer.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Kunstmaan\AdminBundle\Form;
+
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * Class MediaTokenTransformer.
+ */
+class MediaTokenTransformer implements DataTransformerInterface
+{
+    /**
+     * @param mixed $content
+     *
+     * @return string
+     */
+    public function transform($content)
+    {
+        $html = utf8_encode("<!DOCTYPE html>
```

Comment:
@mlebkowski it does work ;) But it's maybe a good solution to adapt it. I will do this later.